src/test/config.sol

```diff
@@ -1612,11 +1612,11 @@ contract Config {
             offboarding:  false
         });
         afterSpell.collaterals["RWA015-A"] = CollateralValues({
-            aL_enabled:   false,
-            aL_line:      0,
-            aL_gap:       0,
-            aL_ttl:       0,
-            line:         2_500_000,
+            aL_enabled:   true,
+            aL_line:      1_280 * MILLION,
+            aL_gap:       50 * MILLION,
+            aL_ttl:       24 hours,
+            line:         0,
```

Comment:
This file is all over the place, but I got a feeling that we were moving towards using `_` separators instead of suffixes:

```suggestion
            aL_enabled:   true,
            aL_line:      1_280_000_000,
            aL_gap:       50_000_000,
            aL_ttl:       24 hours,
            line:         0,
```

This is optional though, feel free to leave as it is.